staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -20,6 +20,27 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
Do we need these consts in v1alpha1 and v1beta1?

Reply:
i removed them all from the external apis. now there's no copy of these consts.